wl-clipboard-rs-tools/build.rs

```diff
@@ -0,0 +1,57 @@
+#[path = "src/bin/wl-copy.rs"]
+#[allow(dead_code)]
+mod wl_copy;
+
+#[path = "src/bin/wl-paste.rs"]
+#[allow(dead_code)]
+mod wl_paste;
```

Comment:
Doesn't this mean that the binaries are compiled an extra time for build.rs? Sounds a bit wasteful; maybe a better idea is to extract the two `Options` structs into separate files and only include those? Not sure how other projects do it.

Reply:
Should be better now